src/style.css

```diff
@@ -1 +1,51 @@
 /* add styles here */
+
+.page {
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+}
+
+.stars__box {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
The `background-image` property is repeated across multiple selectors for `.stars--2 .stars__star`, `.stars--3 .stars__star`, etc. Consider using a more efficient method to apply these styles, such as combining selectors that share the same properties to reduce code duplication. For example, you can write a single selector like `.stars--2 .stars__star:nth-child(-n + 2), .stars--3 .stars__star:nth-child(-n + 3), ... { background-image: url(images/star-active.svg); }`.